### PR TITLE
Add auto-rendered product layout

### LINKED ISSUE #148: Add auto-rendered product layout
Add auto-rendered product layout where the product data is rendered based on [product drop](https://www.voog.com/developers/markup/ecommerce-objects/product) data.

It should look like current "Product" layout. Use new [image](https://www.voog.com/developers/markup/tags/image) or [image_data](https://www.voog.com/developers/markup/tags/image-data) tag for product cover image.

It content type must be `product`:

```json
{
  "content_type": "product",
  "component": false,
  "file": "layouts/auto_generated_product.tpl",
  "layout_name": "auto_generated_product",
  "title": "Auto-generated product"
}
```

See also buy button rendering example in [Nuuk design](https://github.com/Voog/design-nuuk/blob/master/components/buy-button.tpl).

## TODO

* [ ] Add "Auto-generated product" layout
* [ ] Check and improve "[product list](https://www.voog.com/support/online-store/product-list-content-area)" widget styles when needed.

### PR DESCRIPTION
**Pull request by PeeterPaal**
Adds auto-rendered product layout.
Also adds a browser hack to enhance image rendering in Chrome based browsers.
Closes #148

### CODE REVIEW

**Review by tanelj — APPROVED** [review]
Nice work!

**Review by pxska — APPROVED** [review]
👍🏻

**Review by tanelj — CHANGES_REQUESTED** [review]
Nice work and works as expected only one JS error:

* [ ] `BgPicker's `el` parameter has to be either a plain DOM node or a jQuery object around a DOM node.` since there has no BG bicker on the page. `Edicy.BgPicker()`, requires first parameter to be present. https://github.com/Edicy/design-boulder/blob/ee52eb67fe2ac4e2227daaa8bd5452cd0e11023c/components/edicy-tools.tpl#L30

File: javascripts/global-application.js [hunk 1]
```diff
@@ -915,6 +915,50 @@ MMCQ = (function() {
...
+      changeProductImagePos();
+    });
+
+    var changeProductImagePos = function () {
+      var productGallery = $('.js-product-gallery');
+      var productImageContentBox = $('.js-content-item-box');
+      var productRightContent = $('.js-product-right-content');
+
+      if ($('.js-buy-btn-content .edy-buy-button-container').length >= 1) {
+        if ($(window).width() < 600) {
```
Comment (pxska) [inline javascripts/global-application.js:937]: This value should be a bit higher (`< 641` looks correct)

**Review by pxska — APPROVED** [review]

**Pull request merged**